.github/actions/diff-uploader/action.yml

```diff
@@ -0,0 +1,15 @@
+name: 'Diff Uploader'
+description: 'Creates a diff between the commit and the latest Quilt Mappings version'
+inputs:
+  github-token:
+    description: 'Github Token'
+    required: true
+  target-src-exists:
+    description: 'If the diff file was generated'
+    required: true
+outputs:
+  result:
+    description: 'Result of the action'
+runs:
+  using: 'node12'
```

Comment:
I think latest supported node is 14. I might be wrong tho

Reply:
This matches the update-branch action, it would be better to updated both in unison.